internal/tcp_keepalive_unix.go

```diff
@@ -1,3 +1,5 @@
+//go:build unix
```

Comment:
This does seem to work for me.  Without any build constraint:

```
$ GOOS=illumos go build ./internal
$ GOOS=android go build ./internal
$ GOOS=ios go build ./internal
$ GOOS=windows go build ./internal
# google.golang.org/grpc/internal
internal/tcp_keepalive.go:48:10: undefined: unix.SetsockoptInt
internal/tcp_keepalive.go:48:38: undefined: unix.SOL_SOCKET
internal/tcp_keepalive.go:48:55: undefined: unix.SO_KEEPALIVE
```

And with the build constraint, the windows build passes.

Reply:
I didn't realize we could build for any OS with GOOS. Thanks.